### ISSUE TITLE
Kotlin interface extends interface missing override modifier (Version: [graphql-java-codegen 4.1.2])

### ISSUE THREAD

**Issue by ntf** [kobylynskyi/graphql-java-codegen#508]
## Issue Description

When generate Kotlin classes, interface that extends interface doesn't not have proper `override` modifier.

## Steps to Reproduce
Similar to https://github.com/kobylynskyi/graphql-java-codegen/commit/4917c4bddcdc9cda78c04ab9491f220e33086c8f

## Expected Result
```
interface BarBar  : Bar{
   override val id: String
}
```
## Actual Result
```
interface BarBar  : Bar{
    val id: String
}
```
`BarBar` hides member of supertype 'Bar' and needs 'override' modifier.

## Your Environment and Setup

* **graphql-java-codegen**: *E.g.: 4.1.2*
* **Build tool**: *E.g.: Gradle*
* **Java tool**: *E.g.: Oracle JDK 15*
* **Mapping Config**: *E.g.:* 

```groovy
graphqlCodegen {
    // all config options:
    // https://github.com/kobylynskyi/graphql-java-codegen/blob/master/docs/codegen-options.md
    outputDir = new File("$projectDir/src/main/graphql")
    modelNameSuffix = "GraphqlModel"
    apiRootInterfaceStrategy = ApiRootInterfaceStrategy.SINGLE_INTERFACE
    apiNamePrefixStrategy = ApiNamePrefixStrategy.FILE_NAME_AS_PREFIX
    generatedLanguage = GeneratedLanguage.KOTLIN
}
```

**Comment by kobylynskyi** [comment]
@jxnu-liguobin could you please work on this? Thanks!

**Comment by jxnu-liguobin** [comment]
> @jxnu-liguobin could you please work on this? Thanks!

Yes

**Comment by jxnu-liguobin** [comment]
@ntf I've submitted a PR to fix it，please take a look.

**Comment by ntf** [comment]
> @ntf I've submitted a PR to fix it，please take a look.

Thank you. Reviewed and LGTM.

**Comment by kobylynskyi** [comment]
Will be released in 4.1.3
